apps/predictions/lib/predictions_pub_sub.ex

```diff
@@ -33,7 +34,7 @@ defmodule Predictions.PredictionsPubSub do
   def subscribe(key, server \\ __MODULE__) do
     StreamSupervisor.ensure_stream_is_started(key)
     {registry_key, predictions} = GenServer.call(server, {:subscribe, key})
-    Registry.register(:prediction_subscriptions_registry, registry_key, key)
+    Registry.register(@subscribers, registry_key, key)
```

Comment:
It's true, this will return predictions before a newly-started stream has a chance to populate the table.
 
This whole sequence is a weird thing; registering, starting the stream (which will populate the table in `Predictions.Store`) and retrieving predictions (queries the current state of that table), are all independent enough things that probably look more interconnected in these few lines of code than they actually are :D 

It is a bit noticeable on the frontend, but I think I will leave a more thoughtful implementation of this for a future PR.